recipe-server/normandy/recipes/migrations/0047_reciperevision_identicon_seed.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2017-08-16 23:58
+from __future__ import unicode_literals
+
+from django.db import migrations
+import normandy.recipes.fields
+
+
+def generate_identicon_seeds(apps, schema_editor):
+    RecipeRevision = apps.get_model('recipes', 'RecipeRevision')
+
+    for recipe_revision in RecipeRevision.objects.filter(identicon_seed=None):
+        recipe_revision.identicon_seed = f'v1:{recipe_revision.recipe.id}'
+        recipe_revision.save()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('recipes', '0046_reset_signatures'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='reciperevision',
+            name='identicon_seed',
+            field=normandy.recipes.fields.IdenticonSeedField(blank=True, null=True, max_length=64),
+        ),
+        migrations.RunPython(generate_identicon_seeds, generate_identicon_seeds),
```

Comment:
The backwards migration should just be an empty function rather than just re-generating the seeds again.